sdk/communication/azure-communication-identity/tests/test_utils.py

```diff
@@ -20,24 +20,27 @@ def get_token_with_custom_expiry(expires_on):
 
     def test_convert_datetime_to_utc_int(self):
         # UTC
-        utc_time_in_sec = _convert_datetime_to_utc_int(datetime(1970, 1, 1, 0, 0, 0, 0, tzinfo=dateutil.tz.tzutc()))
+        utc_time_in_sec = _convert_datetime_to_utc_int(datetime(1970, 1, 1, 0, 0, 0, 0, tzinfo=timezone.utc))
         assert utc_time_in_sec == 0
         # UTC naive (without a timezone specified)
         utc_naive_time_in_sec = _convert_datetime_to_utc_int(datetime(1970, 1, 1, 0, 0, 0, 0))
         assert utc_naive_time_in_sec == 0
-        # PST is UTC-8
+        # PST is UTC-8 (Vancouver/Pacific Time)
+        pst_tz = timezone(timedelta(hours=-8))  # PST is UTC-8
```

Comment:
nit for timezone info
```suggestion
        pst_tz = ZoneInfo("America/Los_Angeles")
```

Reply:
tests are failing with ZoneInfo as zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key America/Los_Angeles'. As per our offline discussion reverting the change for using the ZoneInfo and keeping timezone format as it is.